model/task/task.go

```diff
@@ -108,11 +108,13 @@ type Task struct {
 
 	BuildId                 string       `bson:"build_id" json:"build_id"`
 	DistroId                string       `bson:"distro" json:"distro"`
+	Container               string       `bson:"container" json:"container"`
```

Comment:
Should the BSON/JSON tags have omitemtpy since it won't be relevant for non-container tasks?